Makefile

```diff
@@ -0,0 +1,12 @@
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+mkfile_dir := $(dir $(mkfile_path))
+TESTS_DIR ?= "$(mkfile_dir)tests"
+RESOURCES_DIR ?= "$(mkfile_dir)resources"
+
+.PHONY: runall
+runall:
+	RESOURCES_DIR=$(RESOURCES_DIR) bats \
+		--verbose-run \
```

Comment:
It seems that `--verbose-run` and `--show-output-of-passing-tests` are new to bats 1.5.0 :/.  OpenSUSE Leap 15.2 has only bats 1.2.0 (Debian Stable the same). I'm ok with bats 1.5.0, but I would like to know the opinion of the others.

Reply:
I'm ok with it. Let me say more: I think it would be great to nixify this projects. That would ensure that everybody has the same versions of the tooling, even across OS's -- even more importantly, it is pinned until we want to bump it, not until it gets bumped in the repo being used.

Reply:
I would be happy with nixification! For starters, just a list of deps and versions on the readme is good for me.

Reply:
Yes, it's necessary to install bats 1.5 manually. I decided to use this version because it allows us to check the command outputs in successful tests with a single command line flag. Which I found useful. 

I'll put  nixify in the TODO list to make that happen. Meanwhile, I'll add more docs about dependencies.